src/app/child-dev-project/attendance/add-day-attendance/roll-call-setup/roll-call-setup.component.spec.ts

```diff
@@ -59,24 +65,19 @@ describe("RollCallSetupComponent", () => {
     expect(component).toBeTruthy();
   });
 
-  it("generates a filled event Note from the RecurringActivity", fakeAsync(() => {
+  it("generates event notes with current user as author", fakeAsync(() => {
```

Comment:
```suggestion
  it("generates event notes with current user as author and details implied from RecurringActivity", fakeAsync(() => {
```

the purpose of this test (although described in weird language) is actually to check that category and subject are taken from activities when generating events. the expectations do not ensure this anymore now